java/BUILD.bazel

```diff
@@ -207,3 +207,28 @@ genrule(
     local = 1,
     tags = ["no-cache"],
 )
+
+java_binary(
+    name = "ray_dist",
+    main_class = "none",
+    runtime_deps = [
+        "//java:org_ray_ray_api",
+        "//java:org_ray_ray_runtime",
+        "//streaming/java:org_ray_ray_streaming-api",
+        "//streaming/java:org_ray_ray_streaming-runtime",
+    ]
+)
+
+genrule(
+    name = "ray_java_pkg",
```

Comment:
we can now change `"$BAZEL_EXECUTABLE" build //java:all --verbose_failures` in `build.sh` to `"$BAZEL_EXECUTABLE" build //java:ray_java_pkg --verbose_failures`.

Reply:
fixed

Reply:
How would pure Java work here?

Reply:
Please see `java/test.sh`